Create pull request view stays empty

GitHub Pull Requests and Issues
v0.55.2022110809

I create a new branch, press the 'Create Pull Request' button, the 'Github Pull Request' view opens, but is empty. 

Happens from time to time, e.g. just now when creating a PR in the in the https://github.com/microsoft/node-request-light

Dev tools console:
```
webview-create-pr-view.js:377 Uncaught TypeError: Cannot read properties of undefined (reading 'availableBaseRemotes')
    at get initialized [as initialized] (webview-create-pr-view.js:377:19653)
    at webview-create-pr-view.js:377:54232
    at Root (webview-create-pr-view.js:377:57061)
    at oh (webview-create-pr-view.js:359:51845)
    at Rj (webview-create-pr-view.js:361:8389)
    at Qj (webview-create-pr-view.js:361:928)
    at Kj (webview-create-pr-view.js:361:831)
    at yj (webview-create-pr-view.js:359:88617)
    at Ig (webview-create-pr-view.js:359:85303)
    at bk (webview-create-pr-view.js:361:16511)
```

In anothe repo I get teh same, but additionally see these error logs ahead if the one listed above. Not sure if they are rela=evant.
There's no `master` branch in this repo, it's `main`
```
[Extension Host] Error: Command failed: "git" merge-base HEAD master
fatal: Not a valid object name master

    at ChildProcess.exithandler (node:child_process:408:12)
    at ChildProcess.emit (node:events:526:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Socket.<anonymous> (node:internal/child_process:451:11)
    at Socket.emit (node:events:526:28)
    at Pipe.<anonymous> (node:net:687:12) (at console.<anonymous> (/usr/share/code-insiders/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:92:141756))
console.ts:137 [Extension Host] fatal: Not a valid object name master
 (at console.<anonymous> (/usr/share/code-insiders/resources/app/out/vs/workbench/api/node/extensionHostProcess.js:92:141756))
notificationsAlerts.ts:40 Error: Command failed: "git" merge-base HEAD master
fatal: Not a valid object name master

    at ChildProcess.exithandler (node:child_process:408:12)
    at ChildProcess.emit (node:events:526:28)
    at maybeClose (node:internal/child_process:1092:16)
    at Socket.<anonymous> (node:internal/child_process:451:11)
    at Socket.emit (node:events:526:28)
    at Pipe.<anonymous> (node:net:687:12)
```

--- a/webviews/common/createContext.ts
+++ b/webviews/common/createContext.ts
@@ -23,7 +23,7 @@ export class CreatePRContext {
 		public onchange: ((ctx: CreateParams) => void) | null = null,
 		private _handler: MessageHandler | null = null,
 	) {
-		this.createParams = vscode.getState();
+		this.createParams = vscode.getState() ?? defaultCreateParams;
 		if (!_handler) {
 			this._handler = getMessageHandler(this.handleMessage);
 		}

